backend/routers/voice.py

```
@@ -87,47 +89,49 @@ async def transcribe_voice(
+            error=None,
         )
-        
+
     except HTTPException:
         raise
     except Exception as e:
         logger.error(f"Error in voice transcription endpoint: {e}", exc_info=True)
-        raise HTTPException(status_code=500, detail=f"Voice transcription failed: {str(e)}")
+        raise HTTPException(
+            status_code=500, detail=f"Voice transcription failed: {str(e)}"
+        )
```

Ruff (0.15.12)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)